octoprint_slicer/static/js/slicer.js

```diff
@@ -499,6 +506,42 @@ $(function() {
             self.renderer.render( self.scene, self.camera );
         };
 
+	self.slicerProperties = ko.observable();
+	self.updateSlicerProperties = function(newSlicersArray) {
+	    if (_.isArray(newSlicersArray) &&
+		newSlicersArray.length > 0) {
+		OctoPrint
+		    .get(OctoPrint.getBlueprintUrl("slicer") +
+			 "slicer/properties")
+		    .then(function (result) {
+			self.slicerProperties(result);
+		    });
+	    } else {
+		self.slicerProperties({});
+	    }
+	}
+	self.slicingViewModel.slicers.subscribe(self.updateSlicerProperties);
+
+	self.sameDevice = ko.computed(function() {
```

Comment:
Thank you for taking my proposal!

You don't have to create a new PR. I believe if you push new commits to your branch `disable_slicing_when_unavailable` the existing PR will get updated.